spec/rails_helper.rb

```diff
@@ -10,6 +10,10 @@
 # Add additional requires below this line. Rails is not loaded until this point!
 require "mock_redis"
 
+# Testing for public_activity gem
+require "public_activity/testing"
+PublicActivity.enabled = false
```

Comment:
Nice idea to turn if off until we need it for testing. It will be creating a lot of records otherwise!